frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -81,73 +79,202 @@ export function Experiment(): JSX.Element {
                             feature_flag_key: newExperimentData?.feature_flag_key,
                             description: newExperimentData?.description,
                         }}
-                        onFinish={(values) => {
-                            setNewExperimentData(values)
-                            nextPage()
-                        }}
+                        onFinish={() => createExperiment(true, runningTime, sampleSize)}
```

Comment:
I don't think we necessarily need a confirmation page since it's just redundant information - we can add the instructions to the JS snippet that exists already below "preview" and the feature flag duplicate key check shouldn't be a problem if we're creating this experiment?

Reply:
https://github.com/PostHog/posthog/pull/7741#discussion_r770595734